sdk/src/wallet/bindings/nodejs/lib/Account.ts

```diff
@@ -394,13 +395,55 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Verifies a message with an ed25519 signature.
+     */
+    async verifyEd25519Signature(
+        signature: Ed25519Signature,
+        message: HexEncodedString,
+    ): Promise<boolean> {
+        const response = await this.messageHandler.callAccountMethod(
+            this.meta.index,
+            {
+                name: 'verifyEd25519Signature',
+                data: {
+                    signature,
+                    message,
+                },
+            },
+        );
+        return JSON.parse(response).payload;
+    }
+
+    /**
+     * Verifies a message with an Evm public key and signature.
+     */
+    async verifySecp256k1EcdsaSignature(
```

Comment:
Should this take a Secp256k1EcdsaSignature ?

Reply:
🤷‍♀️

Reply:
Probably

Reply:
I dunno. It requires making a type somewhere, and I don't know where to put it. The other one is in types, which makes sense because it's part of a block. But this one isn't.

Reply:
Wait I'm confused, you added the type, right? Why not use it?

Reply:
I think I added it to the bindings somewhere, but not in rust.

Reply:
Well yeah, this is a TS file, not rust

Reply:
Done